gap/attr.gi

```diff
@@ -1653,6 +1653,75 @@ function(D)
     return C;
 end);
 
+# Compute for a given rotation system the facial walks
+InstallMethod(FacialWalks, "for a digraph and a list",
+[IsDigraph, IsList],
+function(D, rotationSystem)
+
+    local FacialWalk, facialWalks, remEdges, cycle;
+
+    if not IsEulerianDigraph(D) then
+        Error("The given digraph is not Eulerian.");
+    fi;
+
+    if not IsDenseList(rotationSystem) or Length(rotationSystem)
+      <> DigraphNrVertices(D) then
+        Error("The given rotation system does not fit to the given digraph.");
```

Comment:
```suggestion
        Error("the 2nd argument (list <rotationSystem>) is not a rotation system for the 1st argument (digraph <D>), expected a dense list of length ", DigraphNrVertices(D), 
        "but found dense list of length ", Length(rotationSystem));
```

Reply:
This is a suggestion for a more detailed error message when the number of items in `rotationSystem` is not correct.